doc-site/docs/troubleshooting/undelivered_messages.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Undelivered messages
+---
+
+When using FireFly in multiparty mode to deliver broadcast or private messages, one potential problem is that of
+undelivered messages. In general FireFly's message delivery service should be extremely reliable, but understanding
+when something has gone wrong (and how to recover) can be important for maintaining system health.
+
+## Background
+
+This guide assumes some familiarity with how
+[multiparty event sequencing](../architecture/multiparty_event_sequencing.md) works.
+In general, FireFly messages come in three varieties:
+
+![FireFly Message Types](../images/firefly_message_types.png "FireFly Message Types")
+
+1. **Unpinned private messages:** private messages delivered directly via data exchange
+2. **Pinned private messages:** private messages delivered via data exchange, with a hash of the message recorded on the blockchain ledger
+3. **Pinned broadcast messages:** messages stored in IPFS, with a hash and reference to the message shared on the blockchain ledger
```

Comment:
I wonder if we want to include that registering an identity is a broadcast message